packages/express/src/middleware.ts

```diff
@@ -127,11 +127,11 @@ export function registerExpressMiddleware<CFG>(
  */
 export function asMiddleware(
   options: MiddlewareBindingOptions = {},
-): BindingTemplate<Middleware> {
```

Comment:
is this for a fix or will the ts compiler detect `Middleware` from return type ?

Reply:
This is a fix for TS compiler to allow `asMiddleware()` to be used with `@bind`. Before this PR, the following code fails to compile as `BindingTemplate<Middleware>` cannot be assigned to `BindingTemplate<unknown>`.

```ts
@bind(asMiddleware('my-group')(
```